CHANGELOG.md

```diff
@@ -17,6 +13,8 @@
 * Importing from JSON without the primary key field defined in the JSON object now throws `IllegalArgumentException`.
 * Now `Realm.beginTransaction()`, `Realm.executeTransaction()` and `Realm.waitForChange()` throw `RealmMigrationNeededException` if a remote process introduces incompatible schema changes (#3409).
 * The primary key value of an object can no longer be changed after the object was created. Instead a new object must be created and all fields copied over.
+* Now `Realm.createObject(Class)` and `Realm.createObject(Class,Object)` take the values from the model's constructor (#777).
```

Comment:
Also what about if you assign a value directly on the field? I _think_ in bytecode that is included first in the default constructor,  but I might remember wrong. In any case we should mention it either way.